Infisical/infisical
# feat: show overridden secrets in overview

# Description 📣

This PR enhances the "secrets overview page" to display overridden personal secret values and introduces an "overridden secret" icon in the overview grid.

Related Issue: [#878](https://github.com/Infisical/infisical/issues/878)

Demo: https://share.vidyard.com/watch/BV36qSqheWdwNdAEpaietE?

## Type ✨


# Tests 🛠️

1. Navigate to secret environment page
2. Create a secret and save
3. Mark the secret as "overridden" and change value of the secret to a personal overridden value
4. Navigate back to the "secret overview" page and the value should indicate overridden and show the overridden value.

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/code-of-conduct). 📝](https://share.vidyard.com/watch/BV36qSqheWdwNdAEpaietE?)
